python_modules/libraries/dagster-mysql/dagster_mysql/schedule_storage/schedule_storage.py

```diff
@@ -105,3 +107,22 @@ def connect(self, run_id=None):  # pylint: disable=arguments-differ, unused-argu
     def upgrade(self):
         alembic_config = mysql_alembic_config(__file__)
         run_alembic_upgrade(alembic_config, self._engine)
+
+    def _add_or_update_instigators_table(self, conn, state):
+        selector_id = state.selector_id
+        conn.execute(
+            db.dialects.mysql.insert(InstigatorsTable)
+            .values(
+                selector_id=selector_id,
+                repository_selector_id=state.repository_selector_id,
+                status=state.status.value,
+                instigator_type=state.instigator_type.value,
+                instigator_body=serialize_dagster_namedtuple(state),
+            )
+            .on_duplicate_key_update(
+                status=state.status.value,
```

Comment:
repository_selector_id is not included here since it's always derivable from selector_id?

Reply:
yeah, that's right.